src/sections/Community/Handbook/projects.js

```diff
@@ -1,34 +1,208 @@
+              <img width="3.5%" src={hawkIcon} /> , community with{" "}
+              <img width="3.5%" src={meshmateicon} /> MeshMates, catch-all org,
+              “home base”. <br />
+              <a href="https://github.com/layer5io">
+                <img className="channels-img" src={github} alt="github" />
+              </a>
+              <ul>
+                <li>
+                  <h4>
+                    <a href="https://layer5.io/">Layer5 :</a>
+                  </h4>{" "}
+                </li>
+                <p>
+                  Layer5 is the official website of the Layer5 community showing
+                  an overview of the Layer5 projects. The different service mesh
+                  landscapes, resources to learn about service mesh, and
+                  communities. <br />
+                  <a href="https://github.com/layer5io/layer5">
+                    <img className="channels-img" src={github} alt="github" />
+                  </a>{" "}
+                </p>
+                <li>
+                  <h4>
+                    <a href="https://layer5.io/projects/image-hub">
+                      Image-Hub :
+                    </a>
+                  </h4>{" "}
+                </li>
+                <p>
+                  Image Hub is a sample application written to run on Consul for
+                  exploring WebAssembly modules used as Envoy filters are
+                  written in Rust. These modules can be used to implement
+                  multi-tenancy or to implement per-user rate-limiting in your
+                  application's endpoints. This application was first
+                  demonstrated at DockerCon 2020. <br />
+                  <a href="https://github.com/layer5io/image-hub">
+                    <img className="channels-img" src={github} alt="github" />
+                  </a>{" "}
+                </p>
+                <li>
+                  <h4>
+                    <a href="https://layer5.io/projects/getnighthawk">
+                      GetNightHawk :
+                    </a>
+                  </h4>{" "}
+                </li>
+                <p>
+                  This Is a Layer 7 (HTTP/HTTPS/HTTP2) performance
+                  characterization tool. Nighthawk is Envoy’s load generator and
+                  is written in C++. Meshery integrates Nighthawk as one of
+                  (currently) three choices of load generators for
+                  characterizing and managing the performance of service meshes
+                  and their workloads. <br />
+                  <a href="https://github.com/layer5io/getnighthawk">
+                    <img className="channels-img" src={github} alt="github" />
+                  </a>{" "}
+                </p>
+              </ul>
+            </p>
+            <p>
+              <h3>
+                <a href="https://layer5.io/service-mesh-management/meshery">
```

Comment:
![logo align](https://user-images.githubusercontent.com/85789734/135705869-c0694ce6-1d5a-4972-b397-570a5c7c129e.jpg)
1. None of the logos are aligned with the text on this page. Also, try moving the logos before the text.
2. Meshery operator, meshsync and meshmate logos should be smaller in size to fit well with the text.